### PR TITLE
About us content update

### PR DESCRIPTION
**Pull request by ethanWallace**
# Summary | Résumé

Update contact us page to separate the about us content into it's own page. Also changed `foundations` pages to be `styles` and all the navigation changes that required.

### CODE REVIEW

**Review by daine — APPROVED** [review]
LGTM

**Review by melaniebmn — CHANGES_REQUESTED** [review]
One small question, otherwise looks great!

File: src/en/installation.md [hunk 1]
```diff
@@ -6,20 +6,22 @@ eleventyNavigation:
...
-Our components are developed with tokens designed for Government of Canada branding and styling to help you ship recognizable, predictable, and accessible products.
+- Simplify your digital delivery.
+- Work in the framework you want to work in.
+- Help create seamless Government of Canada service experiences.
 
-Changes to GC Design System (GCDS) come in a single version update with tokens, components, and non-breaking changes to existing ones. GC Design System is independent of the Web Experience Toolkit (WET) and GCWeb.
+## How our updates work
 
-Install GC Design System and try it out!
+Changes to GC Design System (GCDS-) come in a single version update with tokens, components, and (usually) non-breaking changes. GC Design System is independent of the Web Experience Toolkit (WET) and GCWeb.
```
[the remark this answers is not in the document]
Comment (ethanWallace) [reply src/en/installation.md]: Good catch

**Review by melaniebmn — APPROVED** [review]
LGTM

**Review by daine — APPROVED** [review]
Looks good!

**Pull request merged**